### ISSUE TITLE
[Content] Migration is pulling in dead content

### ISSUE THREAD

**Issue by austin-schaefer** [newrelic/docs-website#143]
## Description

The migration script is pulling in content from somewhere that no longer exists. For example, these pages:

* https://www.docs-preview.newrelic.com/docs/servers/new-relic-servers-windows/installation-configuration/installing-servers-windows
* https://www.docs-preview.newrelic.com/docs/servers/servers-pages/servers-ui/new-relic-servers-overview-page

These docs were removed/EoLed a long time ago and redirect to Infra docs.

## Steps to Reproduce

1. Go to https://www.docs-preview.newrelic.com/
2. Click on any of the links in the Servers category

## Expected Behavior

Redirect to appropriate destination

## Additional context

It's possible these pages are still hanging out somewhere in Drupal as "dummy docs," in which case it would be good to have @roadlittledawn help investigate/resolve.

## Acceptance Criteria
* [ ] Assess whether or not a document is a "dummy doc"
* [ ] Redirects are set up to replicate the behavior of "dummy docs"
* [ ] "Dummy docs" files are discarded and _not_ turned into MDX files

**Comment by jerelmiller** [comment]
@austin-schaefer we are currently pulling that data from the docs endpoints: `/api/migration/content/<type>/list`. As you mentioned, we likely need to do a little work on the Drupal side to ensure these "dead" docs don't get synced as real pages.

**Comment by roadlittledawn** [comment]
we're pulling from dev, so that may be the cause here. dev is behind prod database. i haven't synced that in...month or more

**Comment by austin-schaefer** [comment]
@roadlittledawn : Hmm, I think these pages were deleted summer 2019-ish, so shouldn't be impacted by a belated sync.

**Comment by roadlittledawn** [comment]
ah those look like dummy docs. dummy docs are going to be hard to detect because they essentially look like a normal node. @austin-schaefer do we think that all dummy docs have text `Redirect to WHATEVER` in their body?

**Comment by roadlittledawn** [comment]
and are all "archived" docs unpublished? i can't remember what the workaround process was for archiving. I _think_ we unpublished them and set a redirect if applicable, right?

**Comment by jpvajda** [comment]
I'd like to propose we fix this in MMF11 as It would be nice to have all migration tasks done as we going into the final stages of the project, @jerelmiller  / @zstix / @roadlittledawn  what do you think?

**Comment by roadlittledawn** [comment]
I believe the remaining work here is mostly dummy docs. There have been several instances of rogue content coming over which I've been able to fix by tweaking Drupal source (wrong URL paths for release notes and landing pages, filtering certain special nodes out of the JSON, etc).

After doing some of the redirect work and looking at dummy docs currently in the repo, I think it's going to be pretty tough to programmatically find and remove dummy docs. Maybe we add to a production readiness checklist / audit after migration but before launch? For example, I was able to search `src/content/docs` for `Redirects to` (a common but not standard thing for a TW to enter for the body content of a dummy doc) and quickly determine what was a dummy doc. One PR for that and if many eyes are on the repo, maybe we catch all the others?
